predeploy.py

```diff
@@ -20,7 +20,7 @@
 
 for f in file_list:
     shutil.copyfile(os.path.join(app_path, f), f)
-    
+
```

Comment:
Would be good to squash these commits into one because it is easier for the reviewer.